amespahdbpythonsuite/tests/test_spectrum.py

```diff
@@ -61,6 +131,8 @@ def test_fit(self, pahdb_theoretical, monkeypatch):
         # Check plotting function.
         monkeypatch.setattr(plt, 'show', lambda: None)
         spectrum.plot()
+        coadded = fit.coadd(weights=fit.weights, average=False)
+        np.testing.assert_array_almost_equal(test_spec, coadded['data'])
```

Comment:
This is not failing on my machine, but fails here for some reason.

Reply:
The test does actually fail on my machine. When I put a breakpoint just before the assert and subtract the the two arrays I get a maximum difference of -6.103515625e-05. Storing coadded['data'] as a .npy and doing a diff with coadded_test_data.npy shows they are different. I suggest you do something like [line 134 above]:

```
coadded = fit.coadd(weights=fit.weights, average=False)
breakpoint()
np.save('out.npy', coadded['data'])
```

and move out.npy to coadded_test_data.npy

Reply:
That is what I did originally to save the coadded_test_data.npy. Keep in mind I have changed the assertion now to: `np.testing.assert_equal(test_spec, coadded['data'])` as they are expected to be identical. It is strange because the test pass on my machine.

Reply:
Weird indeed. It could be something specific to your system's Python, e.g., 32 bit, endian, representation of null, etc. Possibly the best way forward is for me to sent you the .npz I get.